tests/function_libs/torch_lib/ops_test_data.py

```diff
@@ -1994,6 +1994,11 @@ def _where_input_wrangler(
         trace_only=True,
         tolerance={torch.float16: (8e-2, 1e-4)},
     ),
+    TorchLibOpInfo(
+        "nn.functional.glu",
+        nn_ops.aten_glu,
+        trace_only=True,
+    ),
```

Comment:
```suggestion
    TorchLibOpInfo("nn.functional.glu", nn_ops.aten_glu),
```

The trace_only option is no longer needed